### PR TITLE
Create Gallery App

### PR DESCRIPTION
**Pull request by fortuna**
This will let us develop the UI components in isolation. Example:

image

You can quickly tweak the input and see if it's behaving correctly. This is also an example of using [LitElement](https://lit-element.polymer-project.org/) in our code.

cc: @JonathanDCohen @mpmcroy

### CODE REVIEW

**Comment by fortuna** [comment]
FYI, I've added the l10n messages.

File: src/server_manager/web_app/gallery_app/main.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import IntlMessageFormat from 'intl-messageformat';
```
Comment (alalamav) [inline src/server_manager/web_app/gallery_app/main.ts:1]: nit: copyright header

File: src/server_manager/web_app/gallery_app/main.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  return (msgId: string, ...args: string[]): string => {
+    const params = {} as {[key: string]: any};
+    for (let i = 0; i < args.length; i += 2) {
+      params[args[i]] = args[i + 1];
+    }
+    if (!messages) {
+      // Fallback that shows message id and params.
+      return `${msgId}(${JSON.stringify(params, null, " ")})`;
+    }
+    // Ideally we would pre-parse ang cache the IntlMessageFormat objects,
```
Comment (alalamav) [inline src/server_manager/web_app/gallery_app/main.ts:28]: typo: s/ang/and

File: src/server_manager/web_app/gallery_app/main.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+async function makeLocalize(language: string) {
+  let messages: {[key: string]: string};
+  try {
+    messages = await (await fetch(`./messages/${language}.json`)).json();
+  } catch (e) {
+    window.alert(`Could not load messages for language "${language}"`);
+  }
+  return (msgId: string, ...args: string[]): string => {
+    const params = {} as {[key: string]: any};
```
Comment (alalamav) [inline src/server_manager/web_app/gallery_app/main.ts:20]: nit: tslint to ignore any?

File: tslint.json [hunk 4]
```diff
@@ -16,7 +16,8 @@
     "label-position": true,
     "new-parens": true,
     "no-angle-bracket-type-assertion": true,
-    "no-any": true,
+    // no-any is a pain when working with untyped dependencies in .js, such as our ui components.
+    // "no-any": true,
```
Comment (alalamav) [inline tslint.json:20]: I'm not sure we want to remove this type check. I realize it's not convenient for JS, but we can use directives to ignore safe use of any.

Comment (fortuna) [reply tslint.json]: Reverted. Previously I had any in the middle of HTML and the directive didn't work. But the new structure works fine now.

**Pull request merged**